jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/WriteFlusher.java

```diff
@@ -652,7 +652,7 @@ public String toStateString()
             case COMPLETING -> "C";
             case CANCEL -> "l";
             case CANCELLING -> "L";
-            case FAILED -> "F";
+            case FAILED -> "f";
```

Comment:
I'd keep them all upper-case:

```suggestion
            case CANCEL -> "L";
            case CANCELLING -> "N";
            case FAILED -> "X";
```

Reply:
Done.